core/src/test/java/org/infinispan/eviction/impl/EvictionWithPassivationAndConcurrentOperationsTest.java

```diff
@@ -45,8 +45,7 @@ protected void assertInMemory(Object key, Object value) {
       CacheLoader<Object, Object> loader = TestingUtil.getFirstLoader(cache);
       assertNotNull("Key " + key + " does not exist in data container", entry);
       assertEquals("Wrong value for key " + key + " in data container", value, entry.getValue());
-      MarshallableEntry<Object, Object> entryLoaded = loader.loadEntry(key);
-      assertNull("Key " + key + " exists in cache loader", entryLoaded);
+      eventually(() -> loader.loadEntry(key) == null);
```

Comment:
Do we need the eventually here? I assumed the entry must be in the data container before the get operation finishes.

Reply:
The activation can be done asynchronously from the load https://github.com/infinispan/infinispan/blob/master/core/src/main/java/org/infinispan/interceptors/impl/PassivationCacheLoaderInterceptor.java#L50